index.js

```diff
@@ -106,7 +127,7 @@ class OError extends Error {
 
     const oError = /** @type{OError} */ (error)
 
-    let stack = oError.stack
+    let stack = oError.stack || '(no stack)'
```

Comment:
Just noting that this is now possibly generating

```
SomeTopLevelError: msg
...
cause by:
   (no stack)
```

It was _broken_ before, so I'd leave it as is (proposed in this PR).

<details>

```js
> const OError = require('.')
> try { throw 'msg-lib' } catch (errLib) { err = new OError('msg-app').withCause(errLib); console.log({errLib, stack: errLib.stack, fullStack: OError.getFullStack(err)}) }
{
  errLib: 'msg-lib',
  stack: undefined,
  fullStack: 'OError: msg-app\n' +
    '    at repl:1:48\n' +
    '    at Script.runInThisContext (vm.js:120:20)\n' +
    '    at REPLServer.defaultEval (repl.js:430:29)\n' +
    '    at bound (domain.js:426:14)\n' +
    '    at REPLServer.runBound [as eval] (domain.js:439:12)\n' +
    '    at REPLServer.onLine (repl.js:758:10)\n' +
    '    at REPLServer.emit (events.js:323:22)\n' +
    '    at REPLServer.EventEmitter.emit (domain.js:482:12)\n' +
    '    at REPLServer.Interface._onLine (readline.js:322:10)\n' +
    '    at REPLServer.Interface._line (readline.js:699:8)\n' +
    'caused by:\n' +
    '    (no stack)'
}
```
</details>